README.md

```diff
@@ -9,55 +9,108 @@ http://52.24.131.202:8999/
 
 **NPM:** https://www.npmjs.com/package/gusto-stratera
 
+1)
+Assigned one leader, gave that person authority and responsibility, and held that person accountable for the quality of the prototype submitted.
 
-## Dependencies
-This is a node.js application which requires you to have node installed. To install node, please visit ```https://nodejs.org/download/``` and download and run the latest version of node.
+[See email](DevelopmentPoolEvidence/team/TechnicalLeadEmail.JPG)
 
-Alternatively, it's available for installation through a number of package managers, for example:
+2)
+Assembled a multidisciplinary and collaborative team that includes at a minimum of two of the labor categories limited to the Development Pool labor categories.    
 
-*chocolatey:*
-```bash
-choco install nodejs.install
-```
+Our Team consists of a DevOps Engineer and a Back-end Developer.  Both team members are experienced in the agile scrum development methodology and in playing the role of scrum master.  The DevOps Engineer plays the role of Scrum Master and also performs some of the development tasks.  The Back-end Developer performs the development quality assurance tasks such as peer reviews.
 
-*yum:*
-```bash
-sudo yum install nodejs npm --enablerepo=epel
-```
+3)
+Used at least five modern and open-source technologies, regardless of architectural layer (frontend, backend, etc.)
 
-Other package manager examples can be found on [joyent's github page](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager "joyent's github page").
- 
+| Technology | Version |
+| ---------- | ------- |
+| centos | 6.5 |
+| node | 0.12.0 |
+| express | 4.12.3 |
+| express-http-proxy | 0.6.0 |
+| istanbul | 0.3.16 |
+| mocha | 2.1.0 |
+| mocha-teamcity-reporter | 0.0.1 |
+| should | 7.0.1 |
+| supertest | 1.0.1 |
+| git |1.9.5 | 
 
-## Running gusto from source
-- once you've downloaded the repo, change directory to the root gusto directory
-- run commands
-	-  ```npm install```
-	-  ```npm run start```
 
-At this point, you can hit gusto routes, for example:
+4)
+Deployed the prototype on an Infrastructure as a Service (IaaS) or Platform as a Service (PaaS) provider, and indicated which provider they used
 
-http://localhost:8999/
+The Gusto application is deployed to the Amazon EC2 (IaaS) and is running on t2.micro CentOS 6.5 servers. The delivery of updates to Gusto is being orchestrated by a TeamCity build server which has been established in EC2 and is also being run on a t2.micro server running CentOS 6.5.
 
-http://localhost:8999/stub/
+Gusto has been mande publicly available at the following dns/ip url:
```

Comment:
Should we use the public DNS as the url to get to the application, to demonstrate that Gusto is running on AWS.  (ec2-52-24-131-202.us-west-2.compute.amazonaws.com:8999)